### ISSUE TITLE
MAD - Foreign proxy war national spirits are not removed after Militärverwaltung Madagaskar won the SMU

### ISSUE THREAD

**Issue by Ikidikid** [TNODevs/TNO-Bug-Tracker#3581]
### Operating System

Windows 10 Enterprise

### HOI4 version

1.14.8

### TNO version

1.6.0

### Do you have other mods in use?

None

### Were you using Steam?

Yes

### Which major expansions do you NOT have?

I have all

### Files

[GER_play.zip](https://github.com/user-attachments/files/16836070/GER_play.zip)

### Describe the bug

After the Militärverwaltung Madagaskar won the Second Malagasy Uprising, foreign proxy war national spirits are not removed.

### Steps to reproduce / Cause of the issue

1. The Second Malagasy Uprising triggered.
2. Malagasy Rebellion has foreign proxy war national spirits.
3. Militärverwaltung Madagaskar won the Second Malagasy Uprising.

### Expected behaviour

Forgot to remove the proxy war national spirits once the war is over.

### Screenshots

![Image_1](https://github.com/user-attachments/assets/d0b8e03b-10de-4a71-b890-437d357ca05c)

![Image_2](https://github.com/user-attachments/assets/4af9acbd-d946-4333-a799-d5d953720184)

**Comment by Jack121Q** [comment]
Fixed. Thank you

**Comment by Marok132** [comment]
> Fixed. Thank you

The issue is relevant for other proxy wars fyi.

**Comment by Jack121Q** [comment]
Yep, the fix I made should remove all instances for every tag. I made it auto-remove when the tag is at peace. Except for Technical Aid since that's still relevant.
